codespell_lib/data/dictionary.txt

```diff
@@ -38049,8 +38049,22 @@ peaseful->peaceful
 peasefully->peacefully
 peases->pieces, pleases, leases,
 peasing->piecing, pleasing, leasing,
+pecent->percent, recent, decent,
 pecentage->percentage
+pecentages->percentages
+pecentile->percentile
+pecentiles->percentiles
+pecents->percents
+pecific->specific, Pacific,
```

Comment:
```suggestion
pecific->specific, pacific,
```
Adjective `pacific` as `peaceful`. In such cases, we drop the uppercase, and codespell will fix on its own as follows:
```
pecific → pacific
Pecific → Pacific
```